operations/commit_queue.go

```diff
@@ -351,12 +356,16 @@ func (p *mergeParams) uploadMergePatch(conf *ClientSettings, ac *legacyClient) e
 		return errors.Wrap(err, "can't generate patches")
 	}
 
+	commits := noCommits
 	if commitCount > 0 {
-		patchParams.Description, err = gitCommitMessages(ref.Branch, p.ref, p.commits)
+		var commitMessages string
+		commitMessages, err = gitCommitMessages(ref.Branch, p.ref, p.commits)
 		if err != nil {
 			return errors.Wrap(err, "can't get commit messages")
 		}
+		commits = fmt.Sprintf(commitFmtString, commitMessages, ref.Owner, ref.Repo, ref.Branch)
 	}
+	patchParams.Description = fmt.Sprintf("%s %s", commitQueuePatchLabel, commits)
```

Comment:
Does having "Commit Queue Merge" at the front break the existing validation that the server does? I thought they cased on having a project name or "Revert" at the front of commits (I don't remember if patch description is the same thing as that)

Reply:
this ticket was blocked for a while on DAG updating their validation, but they just updated it to accommodate the change